content-src/components/TopSites/TopSites.js

```diff
@@ -43,7 +44,7 @@ const TopSites = React.createClass({
           return (<div key={site.lastVisitDate || i}>
             <a onClick={() => this.onClick(i)} className={classNames("tile", {active: isActive})} href={site.url}>
               <SiteIcon className="tile-img-container" site={site} faviconSize={32} showTitle />
-              <div className="tile-close-icon" onClick={(ev) => {
+              <div hidden={site.type === FIRST_RUN_TYPE} className="tile-close-icon" onClick={(ev) => {
```

Comment:
Does this need a `ref="delete"` like https://github.com/mozilla/activity-streams/pull/547/files#diff-b0472e345878c3704eb056b33862e4d4R71 above?